pyanaconda/isys/__init__.py

```diff
@@ -99,6 +99,27 @@ def isLpaeAvailable():
 
     return False
 
+def get_cpu_model():
+    """Get the cpu model number."""
+    with open("/proc/cpuinfo", "r") as f:
+        for line in f:
+            name, value = line.split(":")
+
+            if name.strip() == "model":
+                return int(value.strip())
+
+    return None
```

Comment:
I hoped that python will have something in the standard library but I wasn't able to find anything :(.

Please add information about output format to the function documentation.

Reply:
Fixed.

Reply:
I think there should be also `an integer value *or None*`. The None part possibility is missing here.

Reply:
It should be fixed now. Thanks!